Taskfile.yml

```diff
@@ -94,7 +94,7 @@ tasks:
       - "clp-py-utils"
       - "init"
       - "job-orchestration"
-      - "log-viewer-webui"
+      - "log-viewer-webui-clients"
```

Comment:
hey @haiqi96 - when you're available, could you help reviewing the Taskfile and CLP packaging script changes in this PR? There are some changes that might be against what we previously discussed (e.g., we wanted to copy the source files into `<clp-root>/build` before building the client assets); however, with some new measures added (e.g., an additional argument passed to `npm run build`), some previous conclusions might no longer apply. Feel free to object.

Reply:
@kirkrodrigues / @haiqi96 : Let me know if I should split the submission into two PRs (one for log-viewer-webui and one for packaging) now that I realize this one is kinda long.